samples/sampler/stories/default/Image.js

```diff
@@ -1,15 +1,15 @@
 import {action} from '@enact/storybook-utils/addons/actions';
-import {object, select} from '@enact/storybook-utils/addons/controls';
+import {select} from '@enact/storybook-utils/addons/controls';
 import {mergeComponentMetadata} from '@enact/storybook-utils';
 import ri from '@enact/ui/resolution';
 import Image, {ImageBase, ImageDecorator} from '@enact/agate/Image';
 
 import css from './Image.module.less';
 
 const src = {
-	'hd':  'http://via.placeholder.com/200x200',
```

Comment:
It seems that the same svg is created, with or without viewBox and class attributes. Could you please check these attributes and remove them?

Reply:
Suggested changes.